scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1465,88 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None, return_indices=False):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance, optional
+            Object containing the left rotation(s). Default value (`None`)
```

Comment:
Ticks are used to highlight an argument, so just None.

Reply:
Thanks.  I didn't know this.